opensearch-project/OpenSearch-Dashboards
# [Workspace]Fix workspace detail UI bugs

### Description


This PR fix a bunch of UI bugs in workspace detail page, here are the list of changes:
- Update toast message on remove association flow
- Fix data source connection selected items not clear after canceled
- Update "Remove data source(s)" to "Remove data association(s)"
- Update collaborator form group title and description
- Make input readOnly instead of disabled
- Reduce the gap between associate button
- Move form error banner inside tab content
- Update hr margin in tab content
- Update gap between tab header and content
- Add copy tooltip for copy workspace id
- Update "Owner" to "Owners"
- Update use case icon in workspace detail page
- Type workspace name in workspace delete workspace modal
- Render `&mdash;` when description is empty

### Issues Resolved




## Screenshot


![image](https://github.com/user-attachments/assets/2d2481fb-9f25-4ac9-8f98-6ea5df70c604)
![image](https://github.com/user-attachments/assets/710685dd-7b7f-40e5-8d36-7d586625044d)
![image](https://github.com/user-attachments/assets/093cace5-77a2-44f5-a4f5-bba58b5934f1)


## Testing the changes


- Clone branch code and run `yarn osd bootstrap --single-version ignore`
- Add below configs in `config/opensearch_dashboards.yml`
```
savedObjects.permission.enabled: true
workspace.enabled: true
uiSettings:
  overrides:
    'home:useNewHomePage': true
opensearchDashboards.dashboardAdmin.users: ['admin']
```
- Run `yarn start --no-base-path`
- Visiting a existing workspace settings page, all the listed UI bugs should be fixed


## Changelog

- fix: [Workspace]Fix workspace detail UI bugs

### Check List
